src/pages/[sourceFileId].tsx

```diff
@@ -0,0 +1,73 @@
+import App from '../App';
+import { GetServerSideProps } from 'next';
+import { gQuery } from '../utils';
+import {
+  GetSourceFileSsrDocument,
+  GetSourceFileSsrQuery,
+} from '../graphql/GetSourceFileSSR.generated';
+import { AppHead } from '../AppHead';
+import { registryLinks } from '../registryLinks';
+
+interface SourceFileIdPageProps {
+  data: GetSourceFileSsrQuery['getSourceFile'];
+  id: string;
+}
+
+const SourceFileIdPage = (props: SourceFileIdPageProps) => {
+  return (
+    <>
+      <AppHead
+        title={
+          props.data?.name
+            ? `${props.data?.name} | XState Visualiser`
+            : `XState Visualiser`
+        }
+        description={
+          props.data?.name ||
+          `Visualizer for XState state machines and statecharts`
```

Comment:
The "Visualizer" in different copies is not consistent - you are using both Visualizer and Visualiser

Reply:
Grah